sdks/python/apache_beam/transforms/util.py

```diff
@@ -671,6 +681,56 @@ def WithKeys(pcoll, k):
   return pcoll | Map(lambda v: (k, v))
 
 
+@experimental()
+@typehints.with_input_types(typehints.KV[K, V])
+class GroupIntoBatches(PTransform):
```

Comment:
Could you add a pydocs comment to this transform. With, what does it do, what is its expectations, and mentioning that its experimental status?

Reply:
Done.